RubberduckTests/Grammar/ResolverTests.cs

```diff
@@ -6713,5 +6711,79 @@ End Function
                 Assert.IsFalse(failedAccesses.Any());
             }
         }
+
+        [Category("Grammar")]
+        [Category("Resolver")]
+        [Test]
+        public void AssertWithoutDebug_NoReferenceToDebugAssert()
+        {
+            var classCode = @"
+Public Function Foo(index As Variant) As Class1
+    Assert
+End Function
+";
+
+            var moduleCode = $@"
+Private Function Test() As Variant
+    Assert
```

Comment:
Does it matter that the `Assert` is undeclared for the test?

Reply:
That is intentional, Otherwise, it would resolve to the more local declaration in favor of `Debug.Assert`, which we place in the VBA library.